src/cryptoadvance/specter/node.py

```diff
@@ -0,0 +1,320 @@
+import json
+import logging
+import os
+
+from .helpers import is_testnet
+from .rpc import (
+    BitcoinRPC,
+    RpcError,
+    autodetect_rpc_confs,
+    detect_rpc_confs,
+    get_default_datadir,
+)
+from .persistence import write_node
+
+logger = logging.getLogger(__name__)
+
+
+class Node:
+    def __init__(
```

Comment:
Can we have some documentation here? Most stuff is pretty obvious but some not, e.g. what data-type is "external_node"? What's the difference between "name" and "alias". We have that distinction somewhere else as well but haven't documented it there either. Let's start with that here.